frontend/src/components/pages/events/EventDetails.tsx

```diff
@@ -346,6 +348,31 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
                     </Typography>
                   </>
                 )}
+                <Button
+                  variant="text"
+                  href={window.URL.createObjectURL(
+                    new File(
+                      [
+                        `BEGIN:VCALENDAR\nCALSCALE:GREGORIAN\nMETHOD:PUBLISH\nPRODID:-//Test Cal//EN\nVERSION:2.0\nBEGIN:VEVENT\nUID:\nDTSTART;VALUE=DATE:${dayjs(
```

Comment:
```suggestion
                        `BEGIN:VCALENDAR\nCALSCALE:GREGORIAN\nMETHOD:PUBLISH\nPRODID:-//UPDATE THIS//EN\nVERSION:2.0\nBEGIN:VEVENT\nUID:\nDTSTART;VALUE=DATE:${dayjs(
```
Do not commit this suggestion, but remember to update `Test Cal` to whatever the appropriate value is for production.